htdocs/backupListing.php

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
```

Comment:
Indenting not consistent on this file. I think generally we've used tabs since that's what MW does. In this file there are sometimes 4-spaces and other times 2-spaces.